src/views/Games.vue

```diff
@@ -49,6 +49,7 @@ let ofw = [
     message:
       "Enter tag climax and highlight local play. Press [Left Analog (hold down)] and press [L] + [R]."
   },
+  { title: "Mario & Sonic at the Olympic Games Tokyo 2020", lang: "enGB" },
```

Comment:
That super thanks you :)

But can you move this into the `cfw` list? the ofw is the list for Native Lan that works natively in games